check-started/routes/index.js

```diff
@@ -0,0 +1,70 @@
+const express = require('express')
+const router = express.Router()
+const path = require('path')
+const fs = require('fs')
+
+const { checkStarted } = require('../controllers/check-started')
+
+/* Health check */
+async function getPing (req, res) {
+  // get build number from /build.txt
+  // get git commit from /commit.txt
+  let buildNumber = 'NOT FOUND'
+  let commitId = 'NOT FOUND'
+  try {
+    buildNumber = await getBuildNumber()
+  } catch (error) {
+
+  }
+
+  try {
+    commitId = await getCommitId()
+  } catch (error) {
+
+  }
+
+  res.setHeader('Content-Type', 'application/json')
+  let obj = {
+    'Build': buildNumber,
+    'Commit': commitId,
+    'CurrentServerTime': Date.now()
```

Comment:
Also - here - I would very much prefer this to be `moment().toISOString()` as I can't work out seconds since the epoch in my head, so dealing in milliseconds is 1000 times harder than something I find impossible 😉

Reply:
@jon-shipley good point! 😄